gitpod-io/gitpod
# [dashboard] make sure all team route under `/t`

## Description


See also https://github.com/gitpod-io/gitpod/issues/15002#issuecomment-1331979553

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/15002#issue-1466345047

## How to test

- Open preview env
- Go to settings -> Personal Access Token, check if navigation elements are correct

❓ What should we test to prevent regression of `/t/<xxx>` and `<xxx>`

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
